decidim-core/app/helpers/decidim/translations_helper.rb

```diff
@@ -11,8 +11,12 @@ module TranslationsHelper
     #
     # Returns a String with the translation.
     def translated_attribute(attribute)
+      return "" if attribute.nil?
+      return attribute unless attribute.is_a?(Hash)
```

Comment:
Should we duck type instead? No strong opinion, just asking.

Reply:
You mean `respond_to?` well we could do that, but `String` responds to [] & other methods present in Hash. What would you check against?

Reply:
Since the previous version of the method would actually call `[]` on string arguments, I was indeed pointing at checking `respond_to?(:[])`, so we can get rid of the `try`'s. But that was probably incorrect, and what we're fixing here?